ui/app/components/form-field.js

```diff
@@ -4,15 +4,27 @@ import { capitalize } from 'vault/helpers/capitalize';
 import { humanize } from 'vault/helpers/humanize';
 import { dasherize } from 'vault/helpers/dasherize';
 
+/**
+ * @module FormField
+ * `FormField` components are field elements associated with a particular model.
+ *
+ * @example
+ * ```js
+ * {{#each @model.fields as |attr|}}
+ *  <FormField data-test-field @attr={{attr}} @model={{this.model}} />
+ * {{/each}}
+ * ```
+ *
+ * @param [onChange=null] {Func} - Called whenever a value on the model changes via the component.
+ * @param attr=null {Object} - This is usually derived from ember model `attributes` lookup, and all members of `attr.options` are optional.
```

Comment:
we should follow up and spec out this whole object - I think we have a ticket for unifying the interface here already, so that's probably the place to do it

Reply:
Agreed! The object is spec'ed out further down in the file on line 34-43. I wanted to include it in the props table, but unfortunately markdown doesn't allow for code blocks inside tables. Instead added an example attr just below the table and above the 'example usage' section.

Reply:
Nice! Will that change get removed the next time the MD file is regenerated from the JS file?

Reply:
Unfortunately it will, and that's a great point to bring up. I filed a ticket to update `gen-story-md` so it only overwrites sections that have changed.